src/twisted/web/http_headers.py

```diff
@@ -97,13 +100,13 @@ def __cmp__(self, other):
             )
         return NotImplemented
 
-    def _encodeName(self, name):
+    def _encodeName(self, name: Union[bytes, str]) -> bytes:
         """
         Encode the name of a header (eg 'Content-Type') to an ISO-8859-1 encoded
         bytestring if required.
 
         @param name: A HTTP header name
-        @type name: L{unicode} or L{bytes}
+        @type name: L{str} or L{bytes}
```

Comment:
Not really part of this review, I guess, but shouldn't we be getting rid of `@type` and `@rtype` in the epytext now? It's redundant when you have typing info in the actual code, no?

Reply:
That's a good idea.

Reply:
Fixed